### ISSUE TITLE
"FIREWALL SETTINGS" Screen Does Not Update Even After Clicking "RELOAD" In Certain Situations

### ISSUE THREAD

**Issue by BADJAG** [wokhansoft/WFN#44]
I stumbled across a minor (perhaps even trivial) issue in the interface. It's probably too unimportant to address considering the limited time that the developers have and the priority given to other much-needed features, but I thought I'd post it here anyway. Here are the steps to reproduce the problem:

1. Open the FIREWALL SETTINGS screen in WFN. Assume that the notifier has been activated earlier by choosing the BLOCK AND PROMPT option for outbound connections.

2. Open the Windows Firewall control panel applet and click on ADVANCED SETTINGS. In the new window that opens, click ACTION > PROPERTIES. Set the OUTBOUND CONNECTIONS of all three profiles (Domain, Private, and Public) to ALLOW (DEFAULT). Now click OK.

3. Immediately open the properties window you just closed by clicking ACTION > PROPERTIES. This time though, go to the PRIVATE PROFILE tab and set its OUTBOUND CONNECTIONS to BLOCK. Do not change the settings of the other tabs. Click OK.

4. Switch back to the FIREWALL SETTINGS screen in WFN. Click RELOAD at the top. The radio buttons in the OUTBOUND remain unchanged, instead of being updated to the new settings that were specified in the properties page of Step 3. They continue to remain unchanged even after restarting WFN.

Incidentally, If you had switched back to WFN's FIREWALL SETTINGS screen immediately after STEP 3 and clicked the RELOAD button, the settings would automatically have been updated to reflect the changes made through the Control Panel.
 
In short, changing the settings via Control Panel twice consecutively seems to break WFN's ability to update its own settings page, even after restarting the program.

**Comment by wokhan** [comment]
Well this issue is pretty old so it might / should have been fixed.
In the meantime, keeping it open to retry following those detailed steps.
Thank you
